### PR TITLE
Test fixes for ppc and VM jenkins slaves

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -579,19 +579,21 @@ run_mpi_tests() {
...
 
-		# Prevent our tests from using installed UCX libraries
-		export LD_LIBRARY_PATH=${ucx_inst}/lib:$LD_LIBRARY_PATH
-
 		MPIRUN="mpirun \
 				-x UCX_ERROR_SIGNALS \
 				-x UCX_HANDLE_ERRORS \
 				-mca pml ob1 \
-				-mca btl vader,self \
+				-mca btl tcp,self \
```
Comment (shamisp) [inline contrib/test_jenkins.sh]: why?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7134/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7139/ for details (Mellanox internal link).

File: contrib/test_jenkins.sh (hunk 1 above)
Comment (bureddy) [reply contrib/test_jenkins.sh]: It is broken since commit https://github.com/open-mpi/ompi/commit/a82f761a4a6670ff83d3c47601eeb26120d83be5

**Pull request merged**